flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -568,6 +585,49 @@ under the License.
 						</configuration>
 					</execution>
 
+					<execution>
+						<id>fat-jar-kafka-example</id>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<shadeTestJar>false</shadeTestJar>
+							<shadedArtifactAttached>false</shadedArtifactAttached>
+							<createDependencyReducedPom>false</createDependencyReducedPom>
+							<transformers>
+								<transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+									<mainClass>org.apache.flink.streaming.examples.kafka.KafkaExample</mainClass>
+								</transformer>
+							</transformers>
+							<finalName>KafkaExample</finalName>
+							<!-- <outputFile>Kafka.jar</outputFile> -->
+							<filters>
+								<filter>
+									<artifact>*</artifact>
+									<includes>
+										<include>org/apache/flink/streaming/examples/kafka/**</include>
+										<include>org/apache/flink/streaming/**</include>
+										<include>org/apache/kafka/**</include>
+										<include>org/apache/curator/**</include>
+										<include>org/apache/zookeeper/**</include>
+										<include>org/apache/jute/**</include>
+										<include>org/I0Itec/**</include>
+										<include>jline/**</include>
+										<include>com/yammer/**</include>
+										<include>kafka/**</include>
+									</includes>
+								</filter>
+							</filters>
+							<artifactSet>
```

Comment:
nit: please reorder `artifactSet` so that both `KafkaExample` and `Kafka10Example` sections are consistent. (either move this to the top, or move `artifactSet` in `Kafka10Example` to the bottom).